### ISSUE TITLE
[Feature Request] Android TV compatible UI

### ISSUE THREAD

**Issue by 32423d** [crackededed/Xtra#179]
Currently this app is optimized for a phone and tablet layout but not a Android TV layout. It would be nice if a Android TV compatible layout was added to the app. Adding this app to the play store wouldn't be necessary for it to run on Android TVs because Android TVs can still sideload .apk files like other Android devices.

**Comment by Empakt** [comment]
I recommend using Zank Remote on your tv and controlling the app with your phone this way, you will have a mouse cursor and it works really well. I have been doing it this way for like 3 months now.
The Zank Remote also has a quick way to launch any installed app on the device you are connected to.

**Comment by MyNameIsTroll** [comment]
I'm having the same problem, I've installed the Xtra application on my fireTV but it's very difficult to navigate using the remote control buttons. Some menu items are inaccessible, you can never get to them with the 4 directional buttons, and the application is very difficult to use in its current state, unfortunately.

**Comment by Empakt** [comment]
My recommendation being the remote control apps like Zank Remote is not at all a solution, but at least a workaround.
I'm happy to report that doing it this way still works great after more than a year.
I also use YouTube ReVanced on my TV the same way with the remote app.

**Comment by MyNameIsTroll** [comment]
Hi @Empakt , Tanks for the workaround solution.
I've just installed the [S0undTV](https://github.com/S0und/S0undTV) application, which is an alternative Twitch client with an interface designed for TV.
I really like Xtra, I think it would be easy to adapt the navigation for TV use, maybe in the future :)

**Comment by heyboots** [comment]
Yeah I'd also like this, S0undTV hasn't been updated in quite some time and it is no longer blocking ads on my Android TV, but Xtra does. Xtra's UI _mostly_ works for  basic stream viewing but there's currently no way to refresh the Live tab without force closing the app and opening it fresh, since touch gestures don't work.

Even just having an option in the settings to close the app completely when hitting the back button on the main page would be sufficient I think, since it will then refresh the list when it is opened again.

**Comment by peat80** [comment]
A tv ui for this app would be indeed great. ♥️

Almost everything works already on tv using remote.
Only the navigation through this tab based ui is hard with just the remote.
